crates/turborepo-lib/src/commands/login.rs

```diff
@@ -91,7 +87,47 @@ fn make_token_name() -> Result<String> {
     ))
 }
 
+fn print_cli_authorized(user: &str, ui: &UI) {
+    println!(
+        "
+{} Turborepo CLI authorized for {}
+
+{}
+
+{}
+
+",
+        ui.rainbow(">>> Success!"),
+        user,
+        ui.apply(
+            CYAN.apply_to("To connect to your Remote Cache, run the following in any turborepo:")
+        ),
+        ui.apply(BOLD.apply_to("  npx turbo link"))
+    );
+}
+
 pub async fn login(base: &mut CommandBase) -> Result<()> {
+    let user_config = base.user_config()?;
+    // If we've already had the user log in, check to see if
+    // the token we found is still valid.
+    //
+    // If the token we found is not valid, go through the process to get
+    // a new token issued.
+    if let Some(token) = user_config.token() {
+        // Try to fetch the user with the token we found to see if we get a good
+        // response back.
+        // TODO: Write response to disk close to / related to token.
+        match base.api_client()?.get_user(token).await {
```

Comment:
`get_user` works here, but I'd just use `get_teams()` as with sso_token and pull that out into a `validate_token` shared function. 

One hiccup we'll have down the line is that if the user did `turbo login --sso-team=<xx>` the token will only be scoped to that team. So this check for `get_user` would be successful, but not actually grant access to other non-SSO teams. So `get_teams()` will be good for that as well (if just to print out which teams the user has access to)